ebib.el

```diff
@@ -4073,8 +4079,27 @@ a special manner."
       ;; interactively, so `pfx' can only be nil if `ebib-edit-field' is called
       ;; from Lisp code.
       (when pfx (ebib-next-field))
+      (ebib--redisplay-field field)
       (ebib--redisplay-index-item field))))
 
+(defun ebib-edit-current-field ()
+  "Edit current field of a BibTeX entry.
+Most fields are edited directly using the minibuffer, but a few
+are handled specially: the `type' `crossref', `xref' and
+`related' fields offer completion, the `annote', `annotation' and
+`abstract' fields is edited as a multiline field, the `keywords'
+field adds keywords one by one, also allowing completion, and the
+\"file\" field uses filename completion and shortens filenames if
+they are in (a subdirectory of) one of the directories in
+`ebib-file-search-dirs'.
+
+With a prefix argument PFX, edit the current field directly,
+without any special treatment.  Exceptions are the \"type\" field
+and any field with a multiline value, which are always edited in
+a special manner."
```

Comment:
The part about the `PFX` argument is still valid, it's just that there's no argument anymore...

Reply:
Sorted, good spot.